app/main/menu.ts

```diff
@@ -89,7 +89,7 @@ class AppMenu {
 			accelerator: process.platform === 'darwin' ? 'Alt+Command+I' : 'Ctrl+Shift+I',
 			click(_item: any, focusedWindow: any) {
 				if (focusedWindow) {
-					focusedWindow.webContents.toggleDevTools();
+					focusedWindow.webContents.openDevTools({mode: 'undocked'});
```

Comment:
How do we handle the toggle behavior here, since this just seems to open devtools.

Reply:
Yes, it opens DevTools in a new window. We can't toggle it. This was the case earlier for Opening devtools for active tab. But, not we are showing devtools for active tab in the same window and for zulip app in a new window, because zulip app devtools are overlapped by the browser views.

Reply:
So, there is no way to toggle behavior like we had before?

Reply:
No, we can't toggle something that opens in a new window. We coudn't toggle the tools for active tab earlier, now we can't toggle for zulip app, and vice versa.